config/routes.rb

```diff
@@ -6,6 +6,7 @@
     namespace :v1 do
       resources :users, only: %i[show create update destroy]
       resources :tokens, only: %i[create]
+      resources :products
```

Comment:
Can we use only param here?

Reply:
Which verb would you leave out?

Reply:
Products uses all methods, but if you think it's better to be more specific I'll add the param